tests/scripts/coverage/coverage-report.sh

```diff
@@ -15,68 +15,66 @@
 # limitations under the License.
 #
 #
-# Script to run test coverage analysis
+# Script to consolidate coverage reports from multiple runs and convert to HTML
```

Comment:
Isn't this obsolete?

Reply:
No, still combines all the outputs from both machines. just in single scipt